src/components/TodoMainApp.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useContext, useMemo, useState } from 'react';
+import { TodosContext } from '../utils/TodoContext';
+import { Filter } from '../types/Filter';
+import { TodoList } from './TodoList';
+import { TodosFilter } from './TodoFilter';
+
+type Props = {
+};
+
+export const TodoMainApp: React.FC<Props> = () => {
```

Comment:
no needs to declare Props type if you have no props 
```suggestion
export const TodoMainApp: React.FC = () => {
```